server/db/Seeder.js

```diff
@@ -15,15 +15,17 @@ class Seeder {
     let adoptablePets = contents.data.adoptablePets;
 
     try{
-      petTypes.forEach(async (petType) => {
-          let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3)';
-          let result = await pool.query(query, [petType.type, petType.imgURL, petType.description]);
-        });
-
-      adoptablePets.forEach( async (pet) => {
-        let idArr = await pool.query('INSERT INTO adoptable_pets (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING id', [pet.name, pet.img_url, pet.age, pet.vaccination_status, pet.adoption_story, pet.available_for_adoption, pet.pet_type_id])
-      })
-     pool.end();
+      for (let i = 0; i < petTypes.length; i++) {
+        let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3)';
+        let result = await pool.query(query, [petTypes[i].type, petTypes[i].imgURL, petTypes[i].description]);
+      }
+
+      for (let i = 0; i < adoptablePets.length; i++) {
+        let query = 'INSERT INTO adoptable_pets (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING id;'
+        let idArr = await pool.query(query, [adoptablePets[i].name, adoptablePets[i].img_url, adoptablePets[i].age, adoptablePets[i].vaccination_status, adoptablePets[i].adoption_story, adoptablePets[i].available_for_adoption, adoptablePets[i].pet_type_id])
+      }
```

Comment:
This looks great! One thing to keep in mind when creating the `data.json` file to store the information of pet_types and adoptable_pets, I noticing that you are hardcoding the `pet_type_id` to use and were running into issues if things are not created in the correct order.  You could include an extra piece of data for each adobptale_pet that has the pet_type_name.  Then you can use that name to run a query and get the primary key of that record on the pet_types table.  

Not something you have to change, but good to be aware of.

Reply:
Thanks! We're a little worried about time so we're not going to change this now, but we'll it in mind.